rust_sodium-sys/build.rs

```diff
@@ -68,15 +68,16 @@ fn main() {
 
     let command = "(New-Object System.Net.WebClient).DownloadFile(\"".to_string() + &url +
                   "\", \"" + &gz_path + "\")";
-    let download_output = Command::new("powershell")
-        .arg("-Command")
+    let mut download_cmd = Command::new("powershell");
```

Comment:
Just commenting to test the new review tool.  Although this diabolical use of `cmd` as opposed to `command` is worth commenting on!